src/containers/sprite-info.jsx

```diff
@@ -25,7 +25,10 @@ class SpriteInfo extends React.Component {
         };
     }
     handleKeyPress (e) {
-        if (e.key === 'Enter') this.handleFlush();
+        if (e.key === 'Enter') {
+            this.handleFlush();
+            e.target.blur();
+        }
```

Comment:
What's the purpose of blurring the input as soon as you press enter? Just curious.. :)